### DISCUSSION TITLE
notify_push and reverse proxy

### DISCUSSION THREAD

**Discussion by k4my4b** [nextcloud/all-in-one#2139] [Questions] [answered]
I noticed the [README.md](https://github.com/nextcloud/notify_push/blob/main/README.md) of [notify_push](https://github.com/nextcloud/notify_push) mentioning reverse proxies and in case of e.g. Nginx the following might be needed?
```
location ^~ /push/ {
    proxy_pass http://127.0.0.1:7867/;
    proxy_http_version 1.1;
    proxy_set_header Upgrade $http_upgrade;
    proxy_set_header Connection "Upgrade";
    proxy_set_header Host $host;
    proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
}
```

**Comment by Zoey2936** [comment, marked as answer]
This done internally: https://github.com/nextcloud/all-in-one/blob/0b15352132ac95ffa63d87ff9c27b2654ff4ffec/Containers/apache/Caddyfile#L15
